dest/js/libraries-main.js

```diff
@@ -0,0 +1,25 @@
+$(function() {
+  var b = $(".expand-collapse-wrap .button"); // set up all expand buttons
+  var w = $(".expand-collapse-wrap"); // close all expand wraps
+  
+  w.height(100); // customize how much to show
```

Comment:
This is not blocking, but I'm not sure how to customize or what this number means.

Reply:
i guess i could put that in the docs